integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaCluster.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.testkafkacluster;
+
+import java.util.Map;
+
+public interface KafkaCluster extends AutoCloseable {
+    /**
+     * starts the cluster.  use #close will stop it again.
```

Comment:
nit: What does the 2nd line mean here?
`use #close will stop it again.`?

Reply:
It was poor english, fixed.